Modules/DataModel/Sources/PocketCastsDataModel/Private/Managers/EpisodeDataManager.swift

```diff
@@ -104,6 +104,30 @@ class EpisodeDataManager {
 
         return loadMultiple(query: "SELECT \(episodeTableName).* FROM \(upNextTableName) JOIN \(episodeTableName) ON \(episodeTableName).uuid = \(upNextTableName).episodeUuid ORDER BY \(upNextTableName).episodePosition ASC", values: nil, dbQueue: dbQueue)
     }
+    
+    func allUpNextEpisodeUuids(dbQueue: FMDatabaseQueue) -> [Episode] {
+        let upNextTableName = DataManager.playlistEpisodeTableName
+        let episodeTableName = DataManager.episodeTableName
+
+        var episodes = [Episode]()
+        dbQueue.inDatabase { db in
+            do {
+                let resultSet = try db.executeQuery("SELECT \(episodeTableName).uuid FROM \(upNextTableName) JOIN \(episodeTableName) ON \(episodeTableName).uuid = \(upNextTableName).episodeUuid ORDER BY \(upNextTableName).episodePosition ASC", values: nil)
+                defer { resultSet.close() }
+
+                while resultSet.next() {
+                    let episode = Episode()
+                    episode.uuid = DBUtils.nonNilStringFromColumn(resultSet: resultSet, columnName: "uuid")
+                    episode.isStub = true
```

Comment:
I was thinking that maybe this could be a simple computed property on `BaseEpisode` like:

```swift
extension BaseEpisode {
    var isStub: Bool {
        title != nil
    }
}
```

However what properties need to be defined versus what is an Optional and can stay that way? So better to leave it as it is.

I'd change this name to something like: `isFilled` or `hasOnlyUUID` — which conveys better what this field is representing. Stub always makes me think about tests. :P

Reply:
> However what properties need to be defined versus what is an Optional and can stay that way? So better to leave it as it is.

Yeah, that was my thinking too, I can't 100% say for sure if an Episode object is "filled" by just looking at its properties.

> I'd change this name to something like: isFilled or hasOnlyUUID — which conveys better what this field is representing. Stub always makes me think about tests. :P

Sounds good, I'll change it to `hasOnlyUuid`.